markdown/en/api/ff-search-feedback.api.md

```diff
@@ -3,20 +3,20 @@ ___
 ### Properties
```

Comment:
Is it possible to make the first column wider? S. screenshot
![to small fst column](https://user-images.githubusercontent.com/42941766/46860094-f2e93080-ce0f-11e8-9c2c-ee14c2fcaa9d.png)

Reply:
Yes.